components/infobox/wikis/starcraft2/infobox_person_custom_shared.lua

```diff
@@ -84,57 +71,67 @@ function CustomPerson.shouldStoreData()
 end
 
 function CustomPerson.nameDisplay()
-	CustomPerson.getRaceData(_args.race or 'unknown')
-	local raceIcon = RaceIcon({'alt_' .. _raceData.race})
+	local raceData = CustomPerson.readFactions(_args.race or Faction.defaultFaction)
```

Comment:
Missing function?

Reply:
i see it at line 101-125 in the same module

(the function allows the "race" All as an indicator for all 3 factions. I do not want to allow it anywhere outside of the Person infoboxes and neither warcraft nor broodwar need it either hence in this module instead adding more overhead in the Faction module)